pkg/kubelet/rkt/rkt.go

```diff
@@ -1108,6 +1108,18 @@ func (r *Runtime) getSelinuxContext(opt *v1.SELinuxOptions) (string, error) {
 	return strings.Join(ctx, ":"), nil
 }
 
+// From the generateName or the podName return a basename for improve the logging with the Journal
```

Comment:
I used a real example in the TestCase suite, you can have a look [here](https://github.com/kubernetes/kubernetes/pull/40007/commits/3841690dbf100f89875a6cac0fed7eb79eea62fa?diff=unified#diff-c1ee495b0340e6848caab4a82aee160bR1963)